<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Dialog</class>
 <widget class="QDialog" name="Add_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok (✓)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CANCELA">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancela (×)</string>
   </property>
  </widget>
  <widget class="QLabel" name="nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="hora">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HORA</string>
   </property>
  </widget>
  <widget class="QLabel" name="pavimento">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PAVIMENTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="subleito">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SUBLEITO</string>
   </property>
  </widget>
  <widget class="QLabel" name="pressao">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PRESSÃO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="get_nome">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="get_pavimento">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="get_subleito">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="get_pressao">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="get_hora">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>PAVIMENTO:
 F – pavimento flexível (asfalto geralmente);
 R – pavimento rígido (concreto geralmente).

 RESISTÊNCIA DO SUBLEITO:

 A – alta;
 B – média;
 C - baixa;
 D - ultra-baixa.

 PRESSÃO MÁXIMA ADMISSÍVEL DOS PNEUS:

 W - alta (sem limite);
  X – média (até 1,5 MPa);
 Y - baixa (até 1,0 MPa);
 Z - muito baixa (até 0,5 MPa)
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
